src/lib/hooks/routing/clientSideSmartOrderRouter/index.ts

```diff
@@ -50,3 +51,49 @@ export async function getQuote(
 
   return { data: transformSwapRouteToGetQuoteResult(type, amount, swapRoute) }
 }
+
+const protocols: Protocol[] = [Protocol.V2, Protocol.V3]
+export async function getClientSideQuote({
+  tokenInAddress,
+  tokenInChainId,
+  tokenInDecimals,
+  tokenInSymbol,
+  tokenOutAddress,
+  tokenOutChainId,
+  tokenOutDecimals,
+  tokenOutSymbol,
+  amount,
+  type,
+}: {
+  tokenInAddress: string
+  tokenInChainId: ChainId
+  tokenInDecimals: number
+  tokenInSymbol?: string
+  tokenOutAddress: string
+  tokenOutChainId: ChainId
+  tokenOutDecimals: number
+  tokenOutSymbol?: string
+  amount: string
+  type: 'exactIn' | 'exactOut'
+}) {
```

Comment:
can you move this up to a type definition?

Reply:
there are a few function signatures here that I think also merit their own type definitions